docker/tester-exec.sh

```diff
@@ -73,7 +73,7 @@ testimg() {
   # Execute with +e to make sure that possibly created log files get
   # renamed, archived, published even when AFT or some of renaming fails
   set +e
-  daft ${DEVICE} ${FILENAME} --record
+  daft ${DEVICE} ${FILENAME} --record --emulateusb
```

Comment:
The improvement is promising but I'd prefer to keep testing on real HW (especially for Joule since it uses the internal eMMC) because that's what users would have too.